docker-compose-aml-web-client.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Banco Santander S.A.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  tcf:
+    container_name: tcf
+    image: tcf-dev
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.tcf-dev
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    ports:
+      - "1947:1947"
+    environment:
+        - TCF_DEBUG_BUILD=${TCF_DEBUG_BUILD:-}
+        - DISPLAY=${DISPLAY:-}
+        - XAUTHORITY=~/.Xauthority
+    volumes:
+      - ./:/project/TrustedComputeFramework
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - ~/.Xauthority:/root/.Xauthority
+    working_dir: "/project/TrustedComputeFramework/tools/build"
+    entrypoint: "bash -c \"\
+        if [ ${MAKECLEAN:-1} != 0 ]; then echo \\\"Clean build\\\" && make clean; fi \
+        && make \
```

Comment:
We created a new Dockerfile because we needed to create a network and open a port. On the README inside the web-client  folder is explained how to publish the web-app to interact with Avalon on a production environment.

Reply:
@pegartillo95 But you are using the dockerfile (/docker/Dockerfile.tcf-dev) which builds the Avalon code. So, no need to build the code in compose file itself.

Reply:
I think it is fixed now. Could you please check again?